src/api/provisionFormOptionsData.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+	provisionFormOptions: 
+	[
+		{
+			services: [
+				{
+					identifier: 'github'
+				}
+			],
+			automationSoftwares: [
+				'ansible'
+			],
+			serverProviders: [
+				'digitalOcean'
+			],
+			distributions: [
+				'debian'
+			],
+			applications: [
+				{
+					identifier: 'Ghost',
+					name: 'Ghost'
+				},
+				{
+					identifier: 'GitLab',
+					name: 'GitLab'
```

Comment:
same thing here, unless `identifier` or `name` is displayed somewhere. Thoughts @AAlvz